mcfly/modelgen.py

```diff
@@ -44,64 +37,12 @@ def generate_models(x_shape,
         Number of classes for classification task
     number_of_models : int
         Number of models to generate
```

Comment:
Mention that it will be balanced.

Reply:
Actually: remove the balancing. So just mention that this should be >= the number of model types.

Reply:
Done.